debian/eos-updater.install

```diff
@@ -1,4 +1,5 @@
 lib/systemd/system/eos-autoupdater.service
+lib/systemd/system/eos-updater-flatpak-installer.service
```

Comment:
I think we also need to add the flatpak dependency to `debian/control` in the relevant places, unless you’ve done that somewhere else?